Tests/test_PDB.py

```diff
@@ -19,6 +19,7 @@
 import unittest
 import warnings
 from Bio._py3k import StringIO
+from Bio.PDB.PDBParser import PDBParser
```

Comment:
More importantly, it clashes with this existing import later on:

```python
from Bio.PDB import PDBParser, PPBuilder, CaPPBuilder, PDBIO, Select
```

This is due to the old ``Bio.PDB`` module not following PEP8 guidelines that module names are lower case, and classes are title case. Unfortunately here ``PDBParser`` is used for both.

I recommend you remove your new import, and then using the existing import you can refer to the class as ``PDBParser.PDBParser`` instead.

(This is what the  QuantifiedCode check is reporting as *Resolve import naming collision*)